application/classes/Ushahidi/Core.php

```diff
@@ -73,6 +73,11 @@ public static function init()
 				and Kohana::$config->load('features.private.enabled');
 		});
 
+		// Intercom config settings
+		$di->set('site.intercomAppToken', function() use ($di) {
+			return Kohana::$config->load('site.intercomAppToken');
```

Comment:
Grouping this into `site` kind of implies that it can change per deployment... it doesn't.
Site is also user editable... this shouldn't be right?